assets/training/finetune_acft_hf_nlp/src/finetune/finetune.py

```diff
@@ -652,11 +730,25 @@ def finetune(args: Namespace):
     else:
         args.model_name_or_path = args.model_name
 
-    # Enable QLoRA finetune for Llama-70B
+    # init deepspeed config params
+    ds_config_json = None
+    ds_stage = None
+    # Deepspeed enabled
+    if args.apply_deepspeed:
+        # set deepspeed file to be used
+        setup_deepspeed(args)
+        if args.deepspeed:
```

Comment:
may be the check is not needed as we are already checking it in L737

Reply:
reverted functionality